lib/utils.js

```diff
@@ -26,5 +28,20 @@ module.exports = {
     defaultValue: function(obj, defaultValue) {
       return obj || defaultValue;
     }
+  },
+  isValidDataType: function(dataType){
```

Comment:
These utils are specific to MSSQL, why aren't they in the scope of the MSSQL class?  Otherwise you'll end up with a bloated lib/utils.